src/ui/style/Button.js

```diff
@@ -0,0 +1,77 @@
+import styled, { css } from "styled-components";
+
+export const StyledButton = styled.button`
+  width: ${(props) => (props.large ? "122px" : "77px")};
+  padding: 8px 16px;
+  border: none;
+  border-radius: 24px;
+
+  font-family: "Red Hat Display";
+  font-weight: 500;
+  font-size: ${(props) => (props.large ? "18px" : "14px")};
+  text-align: center;
+
+  cursor: pointer;
+
+  &:hover {
+    background-color: var(--primary-green);
+    color: var(--primary-black);
+  }
+
+  @media (min-width: 768px) {
+    width: ${(props) => (props.large ? "182px" : "125px")};
+    padding: 12px 32px;
+  }
+
+  ${(props) => {
+    switch (props.buttonType) {
+      case "primary":
+        return css`
+          background-color: var(--primary-black);
+          color: var(--primary-white);
+
+          &:disabled {
+            background-color: var(--grey-30);
+            color: var(--grey-5);
+          }
+        `;
+      case "secondary":
+        return css`
+          background-color: var(--primary-white);
+          color: var(--primary-black);
+          border: 1px solid var(--primary-black);
+
+          &:hover {
+            border: 1px solid var(--primary-green);
+          }
+
+          &:disabled {
+            background-color: var(--primary-white);
```

Comment:
it can be a better practice to keep only one primary color on your project. do you think we can rename these variables and define a single primary color in your case?
e.g. we define `green` as the primary and keep `black` and `white` just as `black` and `white` themselves.